src/Moryx/Moryx.csproj

```diff
@@ -15,7 +15,10 @@
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" />
-    <PackageReference Include="Microsoft.Extensions.Logging"/>
+    <PackageReference Include="Microsoft.Extensions.Logging" />
+	  
+	<PackageReference Include="Grpc.Net.Client" />
+	<PackageReference Include="Grpc.Core" />
```

Comment:
MORYX does not have any package references except dotnet and Newtonsoft (should be replaced by System.Text.Json later). If this should really part of the MORYX-Framework, this should be part of a separate package to keep the dependencies as minimal as possible